nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    
</pre><p>Can you please show how to load the saved model and make predictions with that here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
> Line #2.    
> Can you please show how to load the saved model and make predictions with that here?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/#comment-1496413067)_

Hello @jmoralez ,

I just added what you requested (via commit commit cbe0c378ab83e5ddaa2450dda59588e763b14e11). However, I am getting an error "KeyError: 0" when trying to pass the Y_df dataframe to the loaded model to make a prediction. Not sure why at this stage, may be a data format issue?

Reply:
<p>I'm not sure what's being loaded here, maybe the loaded_model expects a tensor instead of a dataframe</p><p> </p>